prod-sample/sample-modules/ssr-frank/0.0.0/src/components/SsrFrank.jsx

```diff
@@ -26,7 +25,12 @@ const SsrFrank = ({ moduleState }) => (
   </div>
 );
 
-SsrFrank.loadModuleData = async ({ store, fetchClient }) => {
+const loadModuleData = async ({ store, fetchClient }) => {
+  const moduleState = store.getState().getIn(['modules', 'ssr-frank']);
+  // If isComplete and data already exists dont run request again
+  if (moduleState.get('isComplete') && moduleState.get('data')) {
```

Comment:
is this necessary ?

Reply:
Do you wish for this to double load on both server and client?

Reply:
true true